appveyor.yml

```diff
@@ -1,5 +1,8 @@
 build: false
 
+cache:
+  - C:\Users\appveyor\AppData\Local\pip\Cache -> appveyor.yml
```

Comment:
if i'm not wrong, this means that the cache will be invalidated if the appveyor yaml file is updated.

Reply:
Yes. This gives us an easy way to invalidate the cache should we need to.